ddmrp_warning/models/ddmrp_warning_definition.py

```diff
@@ -23,6 +24,26 @@ class DdmrpWarningDefinition(models.Model):
         default="mid",
     )
     active = fields.Boolean(default=True)
+    warning_domain = fields.Char(
+        string="Buffer Applicable Domain",
+        default=[],
```

Comment:
a list as default for char?

Reply:
I know strange, but it works as I get the proper value in the db.
I changed it to a string anyway.